sub/coq

```diff
@@ -1 +1 @@
-Subproject commit b3bd2696c31ad2cb544f3436ddb5a237fe7fa6fe
+Subproject commit 76d6a8cfb0448ade98e1a7abc92ff1bf5075fe8f
```

Comment:
I will try to correct.

2015-10-09 16:19 GMT-04:00 Benedikt Ahrens <notifications@github.com>:

> In sub/coq
> <https://github.com/UniMath/UniMath/pull/185#discussion_r41673646>:
>
> > @@ -1 +1 @@
> > -Subproject commit b3bd2696c31ad2cb544f3436ddb5a237fe7fa6fe
> > +Subproject commit 76d6a8cfb0448ade98e1a7abc92ff1bf5075fe8f
>
> Do not change the Coq commit to an older one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UniMath/UniMath/pull/185/files#r41673646>.
>